### ISSUE TITLE
How to improve efficiency when sess.run

### ISSUE THREAD

**Issue by zysNLP** [tensorflow/tensorflow#27415]
When I test my model using Tensorflow, I wonder how to imporve the speed (or reduce running time) when I execute  session like:

with tf.Session(graph=graph) as sess:
    model.saver.restore(sess, conf["init_model"])
    feed = { 
        model.data: mydata,  
        model.label: mylabel
        }
    scores = sess.run(model.logits, feed_dict = feed)
    losses = sess.run(model.loss, feed_dict = feed)

As my model size and number of parameter being fixed (after training), How can I impove the speed when I test model with only little batch (even one batch would be slow) to get scores and losses.
I wonder if taking more GPUs could be help. I am not sure.
Does there any other idea to solve it? Thank you !

**Comment by muddham** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new/choose). Thanks!

**Comment by jvishnuvardhan** [comment]
@zysNLP Please take a look at the [tutorial](https://www.tensorflow.org/tutorials) section in TF website and most importantly look at the tutorials under "ML at production scale". There are several approaches to optimize ML model and there is vast amount of information available (TF website, stackoverflow, github, medium, kaggle, .......).
I am closing the issue here as it is not related to Bug or performance related to TF. GitHub repo is mainly for addressing bugs in installation and performance. Thanks!

**Comment by zysNLP** [comment]
OK, sorry and thankyou very much!
